### DISCUSSION TITLE
Release 15.2.1 — Comunicación Dialogs

### DISCUSSION THREAD

**Discussion by Leycarr** [Telefonica/mistica-design#1743] [Proposals]
- [ ] Actualizar propiedades del componente de los Alert/Dialog/Confirm.
- [ ] Propiedades actuales no permiten añadir icono en los componente alert y confirm. 
La solución es usar a dialog , para mantener las funciones del diseño y tampoco tiene la función de desactivar uno de los botones o omitirlo, y 3 botones que tienen la misma función (omitir la información dándola por entendido) y confunde al usuario. Necesitaríamos que uno de los botones junto el dismiss se pudieran desactivar para no confundir las funciones.
 
 Adjuntamos captura de como era antes y como es ahora.
![imagen](https://github.com/Telefonica/mistica-design/assets/126065149/f8f3c6a8-434c-4c47-a7ba-5a81331515e1)
![imagen (1)](https://github.com/Telefonica/mistica-design/assets/126065149/69b8f1f7-924c-4f0a-b822-3fc25eb34d2c)

**Comment by aweell** [comment]
Buenas @Leycarr 

Voy a dividir aquí lo que son decisiones de diseño de lo que es un bug de web.

### Bug de web

> tampoco tiene la función de desactivar uno de los botones o omitirlo, y 3 botones que tienen la misma función (omitir la información dándola por entendido

Según las specs el dialog puede configurarse para tener desde ninguna a 3 acciones. 

Esto se resuelve en: https://github.com/Telefonica/mistica-web/pull/1093

### Iconos en alert / confirm

> Propiedades actuales no permiten añadir icono en los componente alert y confirm.

Estos componentes están pensando para ir en consonancia tanto como con las implementaciones nativas como con cómo se comporta en el browser por lo que no se plantea que lleven iconos.

### Dismiss del dialog

> Necesitaríamos que uno de los botones junto el dismiss se pudieran desactivar para no confundir las funciones.

En el caso del dismiss se tomó esa decisión ya que no ocurre que en todos los casos las acciones del dialog descarten el mismo. Que exista la posibilidad de configurar ese dismiss podría ser una discussion en este repositorio.

**Reply by ipozuelo** [reply]
Buenas, desde el equipo de desarrollo no vemos la opción de cómo poder quitar los botones de aceptar y cancelar. Hemos recreado el mismo ejemplo que hay en el storybook y no ha funcionado. Estamos en la versión 15.3.0. Podéis mandarnos un ejemplo real para ver como utilizar el componente adecuadamente? Este es una prueba que hemos realizado sin éxito.
![image](https://github.com/Telefonica/mistica-design/assets/127224788/e14541b1-b686-4228-bf45-396804f6a1d8)

**Reply by aweell** [reply]
@pladaria FYI

**Reply by ipozuelo** [reply]
Buenas, no se si han podido echar un vistazo a esto.

**Reply by atabel** [reply]
@ipozuelo Que version de mistica estais usando?
Esto se arreglo en la release 15.4
https://github.com/Telefonica/mistica-web/pull/1093#issuecomment-2096404040

**Reply by ipozuelo** [reply]
La 15.3 que es la última de hace dos semanas. Actualizo y te comento

**Reply by yceballost** [reply]
@ipozuelo puedes confirmar si esto quedó solucionado?

**Comment by yceballost** [comment]
Cierro! Entendemos que quedó solucionado
